secrets/fake/fake.go

```diff
@@ -0,0 +1,40 @@
+package fake
+
+import (
+	"errors"
+	"github.com/libopenstorage/openstorage/secrets"
+)
+
+var (
+	// ErrNotImplemented default secrets in OSD
+	ErrNotImplemented = errors.New("Not Implemented")
+)
+
+type Fake struct {
```

Comment:
this struct can be a private one. I don't see any reason why it needs to be public.